lib/cartopy/mpl/geoaxes.py

```diff
@@ -417,6 +417,34 @@ def coastlines(self, resolution='110m', color='black', **kwargs):
                                                       resolution, **kwargs)
         return self.add_feature(feature)
 
+    def tissot(self, rad_km=5e5, n_samples=80, lon_n=10, lat_n=6, **kwargs):
+        """
+        Adds Tissot's indicatrices to the current axes with the geodesic class.
+
+        Kwargs:
+
+            * rad_km - The radius in km of the the circles to be drawn.
+
+            * n_samples - Integer number of points sampled around the
+              circumference of each circle.
+
+            * lon_n - Number of circles in the longitudinal direction.
+
+            * lat_n - Number of circles in the latitudinal direction.
+
```

Comment:
Ahh okay. From a user perspective though, it doesn't seem like it is very easy to find exactly which `**kwargs` can be used. The doc for `ShapelyFeature` just says:

`Kwargs:
    Keyword arguments to be used when drawing this feature. `

How is someone supposed to find a list of usable `**kwargs`? Am I missing something?